components/GlobalIANavigationBar/components/Menu.module.scss

```
@@ -36,26 +37,29 @@ $menu__square-size: $ca-grid * 2;
 .menu {
   $menu__arrow-size: $ca-grid / 4;
 
-  align-items: center;
+  @include ca-position($top: calc(100% + #{$ca-grid * 0.75}), $end: 0);
 
+  align-items: center;
   position: absolute;
   height: 100%;
-  top: calc(100% + #{$ca-grid * 0.75});
-  right: 0;
   z-index: $ca-z-index-dropdown;
 
   // menu arrow
   &::before {
     content: '';
     position: absolute;
-    right: calc(#{$menu__square-size / 2} - #{$menu__arrow-size});
     top: -0.3rem;
-
+    left: calc(#{$menu__square-size / 2} - #{$menu__arrow-size});
```

```suggestion
    right: calc(#{$menu__square-size / 2} - #{$menu__arrow-size});
```

Should this be `right:` not `left:`?